canonical-web-and-design/ubuntu.com
# Fix a load of link errors across the site

## Done
Fix all links on live

## QA
- Check all the new links work
- Check it covers all errors reported by [the latest links on live action](https://github.com/canonical-web-and-design/ubuntu.com/runs/2525242958?check_suite_focus=true)
